### ISSUE TITLE
new module: macsima2mc

### ISSUE THREAD

**Issue by luiskuhn** [nf-core/modules#5253]
Module for running the conversion of MACSima images to MCMICRO input https://github.com/SchapiroLabor/multiplex_macsima/

The aim is to include this in the nf-core/mcmicro pipeline: https://github.com/nf-core/mcmicro

### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

**Comment by famosab** [comment]
@luiskuhn @mbruhns Is this still on your list?

**Comment by VictorDidier** [comment]
@famosab It is in mine.  I will start working on it.

**Comment by luiskuhn** [comment]
Yes, let's wrap it up, perhaps @mbruhns can help here. I am happy to help out
